src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -46,7 +46,8 @@ define(function (require, exports, module) {
     function _mapError(err) {
         if (!err) {
             return null;
-        }
+         }
```

Comment:
why is there an extra space here now? `npm test` should have flagged this as being the wrong indentation.